forem/forem
# Add missing aria label for campaign close button

## What type of PR is this? (check all applicable)


## Description
Fixes an accessibility issue with campaign close button

## Related Tickets & Documents
Closes #11859

## QA Instructions, Screenshots, Recordings

1. Login as an administrator with edit rights to the configuration of Forem.
2. Create a campaign by following the instructions in the [admin guide](https://forem.gitbook.io/forem-admin-guide/misc/customizing-with-campaigns-+-display-ads).
3. Test with a screen reader by focusing on the close button. A tool like Voice Over on Mac should work as a screen reader. The screen reader should say, "Close campaign banner."

### UI accessibility concerns?

This PR fixes an accessibility issue with the close button.

## Added tests?

- [x] No, and this is why: Tested this manually with a screen reader.

## Added to documentation?

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
